### DISCUSSION TITLE
Continue response doesn't work

### DISCUSSION THREAD

**Discussion by IcyIntuition** [open-webui/open-webui#3950] [General]
# Bug Report

## Description

**Bug Summary:**

The continue response button doesn't work as expected. Letting the LLM continue an answer in the playground also doesn't seem to work.

**Steps to Reproduce:**

Edit a `assistant` type message and try to let the LLM continue it.

**Expected Behavior:**

The LLM should continue the `assistant` message.

**Actual Behavior:**

The LLM only uses previous messages to generate the completion and ignores the edited `assistant` message.

## Environment

- **Open WebUI Version:** 0.3.10
- **Ollama (if applicable):** 0.2.5

- **Operating System:** Arch Linux
- **Browser (if applicable):** Firefox 128

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

Docker package.

## Additional Information

I did check the Ollama debug logs and I couldn't find the edited message in there, so I guess that this is an issue in Open WebUI and not Ollama.

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by tjbck** [comment]
<img width="572" alt="image" src="https://github.com/user-attachments/assets/623b0e0d-f452-4907-8a84-7f2630d6f1ce">


Payload seems to be correctly sending the message.

**Comment by IcyIntuition** [comment]
I do see my edited message in the API call to Open WebUI's `completions` endpoint, but it is not passed to Ollama.

Here is an example from the playground. I manually added the `The` at the start of the assistant message, which is ignored:
![grafik](https://github.com/user-attachments/assets/0914cefd-7270-4cc4-b688-d40cf9f4132c)

The request to Open WebUI's `completions` endpoint:
![grafik](https://github.com/user-attachments/assets/7cdd33e9-75c1-4164-a7ea-720229df5b50)

And the corresponding log entry from Ollama:
```
VERB [log_server_request] request | request="{\"cache_prompt\":true,\"frequency_penalty\":0,\"image_data\":null,\"main_gpu\":0,\"mirostat\":0,\"mirostat_eta\":0.1,\"mirostat_tau\":5,\"n_keep\":24,\"n_predict\":81920,\"penalize_nl\":true,\"presence_penalty\":0,\"prompt\":\"<|start_header_id|>user<|end_header_id|>\\n\\nTell me a random fun fact about the Roman Empire<|eot_id|><|start_header_id|>assistant<|end_header_id|>\\n\\n\",\"repeat_last_n\":64,\"repeat_penalty\":1.1,\"seed\":-1,\"stop\":[\"<|start_header_id|>\",\"<|end_header_id|>\",\"<|eot_id|>\"],\"stream\":true,\"temperature\":1,\"tfs_z\":1,\"top_k\":40,\"top_p\":1,\"typical_p\":1}\n" response="" tid="125413888100032" timestamp=1721237026
```

**Comment by Sureiya** [comment]
Can confirm this has also been happening to me for the past few weeks.

**Comment by TomerGamerTV** [comment]
Bump

**Comment by billymambo** [comment]
bump
